src/admin_websocket.rs

```diff
@@ -57,15 +57,15 @@ impl AdminWebsocket {
     ///
     /// As string `"localhost:30000"`
     /// As tuple `([127.0.0.1], 30000)`
-    pub async fn connect(socket_addr: impl ToSocketAddrs) -> Result<Self> {
```

Comment:
`anyhow` is still here but we should be trying to get rid of it. It's horribly hard to handle in consumer libraries.